packages/website/docs/createQuerySuggestionsPlugin.md

```diff
@@ -54,35 +80,39 @@ autocomplete({
 });
 ```
 
-## Import
-
-```ts
-import { createQuerySuggestionsPlugin } from '@algolia/autocomplete-plugin-query-suggestions';
-```
+To see it in action, check [this demo on CodeSandbox](https://fzb4m.csb.app/).
 
-## Params
+## Parameters
 
 ### `searchClient`
 
 > `SearchClient` | required
 
+The initialized Algolia search client.
+
 ### `indexName`
 
 > `string` | required
 
+The index name.
+
 ### `getSearchParams`
 
-> [`() => SearchParameters`](https://www.algolia.com/doc/api-reference/search-api-parameters/)
+> `() => SearchParameters`
+
+A function returning [Algolia search parameters](https://www.algolia.com/doc/api-reference/search-api-parameters/).
 
 ### `transformSource`
 
 > `(params: { source: AutocompleteSource, onTapAhead: () => void })`
 
-#### Example
+A function to transform the source based on the Autocomplete state.
```

Comment:
It's not necessarily based on the state, you might just want to edit the provided templates.

```suggestion
A function to transform the provided source.
```